src/QsCompiler/CSharpGeneration/CSharpGeneration.fsproj

```diff
@@ -24,4 +25,11 @@
   <ItemGroup>
     <ProjectReference Include="..\RoslynWrapper\RoslynWrapper.fsproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Tests.Microsoft.Quantum.CSharpGeneration$(PublicKeySuffix)</_Parameter1>
```

Comment:
(personal curiosity) If needed then where can one see the description of identifiers like this one - `<_Parameter1>`?

Reply:
That's a good question. I generally haven't found a really good answer for where to look for msbuild features (there is some good docs, but rarely for the things I was looking for). On this particular topic, I would search online for defining InternalsVisibleTo (or any assembly attribute) in the project file, and might e.g. find the [blog post](https://www.meziantou.net/declaring-internalsvisibleto-in-the-csproj.htm)
or the [StackOverflow answer](https://stackoverflow.com/questions/42810705/visual-studio-2017-and-the-new-csproj-internalsvisibleto), which by the way has a comment "Not sure this approach is documented since it is rather customization than common practice."